main.go

```diff
@@ -413,10 +417,10 @@ func main() {
 			Usage: "sets the indexer advertisement url",
 			Value: cfg.Node.IndexerURL,
 		},
-		&cli.IntFlag{
-			Name:  "indexer-tick-interval",
-			Usage: "sets the indexer advertisement interval in minutes",
-			Value: cfg.Node.IndexerTickInterval,
+		&cli.StringFlag{
+			Name:  "indexer-advertisement-interval",
+			Usage: "sets the indexer advertisement interval",
```

Comment:
please a description here that the value is in seconds. 

`sets the indexer advertisement interval in seconds.`